ui-components/src/page_components/sign-in/FormSignInMFACode.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import {
+  AppearanceStyleType,
+  Button,
+  Field,
+  Form,
+  FormCard,
+  Icon,
+  t,
+  SiteAlert,
+  FormSignInErrorBox,
+} from "@bloom-housing/ui-components"
+import { FormSignInNetworkError, FormSignInControl } from "./FormSignIn"
+import { EnumRequestMfaCodeMfaType } from "@bloom-housing/backend-core/types"
+
+export type FormSignInMFACodeProps = {
+  control: FormSignInControl
+  onSubmit: (data: FormSignInMFACodeValues) => void
+  networkError: FormSignInNetworkError
+  mfaType: EnumRequestMfaCodeMfaType
+}
+
+export type FormSignInMFACodeValues = {
+  mfaCode: string
+}
+
+const FormSignInMFACode = ({
+  onSubmit,
+  networkError,
+  control: { errors, register, handleSubmit },
+  mfaType,
+}: FormSignInMFACodeProps) => {
+  const onError = () => {
+    window.scrollTo(0, 0)
+  }
+
+  return (
+    <FormCard>
+      <div className="form-card__lead text-center border-b mx-0">
+        <Icon size="2xl" symbol="profile" />
+        <h2 className="form-card__title">{t("nav.signInMFA.verifyTitle")}</h2>
+        <p className="form-card__sub-title">
+          {mfaType === EnumRequestMfaCodeMfaType.sms
+            ? t("nav.signInMFA.haveSentCodeToPhone")
+            : t("nav.signInMFA.haveSentCodeToEmail")}
+        </p>
+      </div>
+      <FormSignInErrorBox errors={errors} networkError={networkError} errorMessageId={"mfa-code"} />
+
+      <SiteAlert type="notice" dismissable />
+      <div className="form-card__group pt-0 border-b">
+        <Form id="sign-in-mfa" className="mt-10" onSubmit={handleSubmit(onSubmit, onError)}>
+          <Field
+            caps={true}
+            name="mfaCode"
+            label={t("nav.signInMFA.code")}
+            validation={{ required: true }}
+            error={errors.mfaType}
+            errorMessage={t("nav.signInMFA.noMFACode")}
+            register={register}
+            dataTestId="sign-in-mfa-code-field"
+          />
+
+          <div className="text-center mt-6">
```

Comment:
<img width="366" alt="Screen Shot 2022-02-08 at 4 52 22 PM" src="https://user-images.githubusercontent.com/65367387/153095414-863e4362-fafb-46ed-9f93-05d30969f33d.png">

I'm not seeing the edit phone number line here